benches/test_agentic_benchmark.py

```
@@ -0,0 +1,247 @@
+    assert "error" in hs.search("q")
+
+
+def test_make_web_backend_default_frozen_and_live_requires_key():
+    parser = B.make_parser()
+    frozen = B.make_web_backend(parser.parse_args(["--offline-self-check"]))
+    assert isinstance(frozen, B.FrozenWebSearch)
+    live = parser.parse_args(["--offline-self-check", "--web-search", "live",
+                              "--web-search-key-env", "DEFINITELY_UNSET_ENV_XYZ"])
+    try:
+        B.make_web_backend(live)
+        assert False, "expected SystemExit when the key env var is missing"
```

Ruff (0.15.18)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)